activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1099,6 +1099,17 @@ def foreign_keys(table_name)
       #
       #   ALTER TABLE "articles" ADD CONSTRAINT fk_rails_58ca3d3a82 FOREIGN KEY ("author_id") REFERENCES "users" ("lng_id")
       #
+      # ====== Creating a composite foreign key
+      #
+      #   # Assuming "carts" table has "(shop_id, id)" as a primary key.
+      #   add_foreign_key :orders, :carts, primary_key: [:shop_id, :id]
+      #
+      # generates:
+      #
+      #   ALTER TABLE "orders" ADD CONSTRAINT fk_rails_6f5e4cb3a4 FOREIGN KEY ("cart_shop_id", "cart_id") REFERENCES "carts" ("shop_id", "id")
+      #
+      # Note: At least one of +column+ or +primary_key+ options must be provided.
```

Comment:
I'm not super clear on why we need to support both, can we support just primary_key and simplify the code a bit?